src/status_im2/contexts/chat/messages/pin/banner/view.cljs

```diff
@@ -1,15 +0,0 @@
-(ns status-im2.contexts.chat.messages.pin.banner.view
```

Comment:
could you please keep this component and just replace `[pinned-banner/pinned-banner` on quo2 on here, because we still want all related subscriptions to be here

Reply:
yeah I can do this, it's for performance reasons?

Reply:
both performance and more clean code

Reply:
done